deploy/examples/import-external-cluster.sh

```diff
@@ -167,6 +168,18 @@ function importConfigMap() {
   fi
 }
 
+function createInputCommadConfigMap() {
+  if ! $KUBECTL -n "$NAMESPACE" get configmap "$EXTERNAL_COMMAND_CONFIGMAP_NAME" &>/dev/null; then
+    $KUBECTL -n "$NAMESPACE" \
+      create \
+      configmap \
+      "$EXTERNAL_COMMAND_CONFIGMAP_NAME" \
+      --from-literal=command="$EXTERNAL_CLUSTER_USER_COMMAND" \
+      else
+    echo "configmap $EXTERNAL_COMMAND_CONFIGMAP_NAME already exists"
```

Comment:
If it already exists, should we assume that this is a re-run with an update? In that case I think the tool should update the existing cm

Reply:
Thats correct we need to patch it.